perf/benchmark/security/generate_policies/README.md

```diff
@@ -15,19 +15,29 @@ To generate specific security policies begin by creating a json file that has th
 {
   "AuthZ":
   {
-    "action":int,           // optional DENY/ALLOW. Default:DENY
-    "numNamespaces":int,    // optional. Default:0
-    "numPaths":int,         // optional. Default:0
-    "numPolicies":int,      // optional. Default:0
-    "numPrincipals":int,    // optional. Default:0
-    "numSourceIP":int,      // optional. Default:0
-    "numValues":int         // optional. Default:0
+    "action":int,                 // optional DENY/ALLOW. Default:DENY
```

Comment:
action should be a string?

Also you can remove the comment of `Default: 0` and `Default: false` in other fields.

Reply:
updated and removed